src/components/App.js

```diff
@@ -1,12 +1,14 @@
 import Display from './Display';
 import ButtonPanel from './ButtonPanel';
+import calculate from '../logic/calculate';
 
 function App() {
   return (
     <div className="Component-App">
       <>
         <Display />
         <ButtonPanel />
+        <calculate />
```

Comment:
You don't need to add this line, you have done well by adding the `calculate` function on line 3.
kindly remove this line.

Reply:
**The following changes were made:**

- Destructure data parameter into total, next, and operation in calculate class.

- I had to leave line 11 in App.js. When I remove it, I get an undefined error. Besides, Adding it is part of the milestone requirements. see screen-shot for reference.

![image](https://user-images.githubusercontent.com/51296741/111083131-2b732400-850c-11eb-9207-3d5a32a3c4fc.png)

Reply:
@enaburekhan 🙂, I get your point.
I think the numbering is confusing, 
let say `Add calculate.js to the App component` is the title
and `import the calculate function is the subtitle.

15 - saying what to do 
16 - saying how to go about it.

kindly remove in the next milestone, i understand the `undefined` error message.